auth-api/tests/conftest.py

```diff
@@ -338,3 +338,13 @@ def publish(self, *args, **kwargs):
             raise CancelledError("This is a mock")
 
     mocker.patch("google.cloud.pubsub_v1.PublisherClient", PublisherMock)
+
+
+@pytest.fixture(autouse=True)
+def mock_has_nsf_or_suspended_membership(monkeypatch):
```

Comment:
Is this needed? do the other unit tests work with a SUSPENDED or NSF ORG? I figured they'd be ACTIVE